docs/Fields.md

```diff
@@ -491,6 +491,14 @@ To change the link from the `<Edit>` page to the `<Show>` page, set the `linkTyp
 </ReferenceField>
 ```
 
+By default, `<ReferenceField>` is sorted by its `source`. To specify another attribute to sort by, set the `sortBy` prop to the according attribute's name.
+
+```jsx
+<ReferenceField label="User" source="userId" reference="users" sortBy="name">
```

Comment:
Not a good example imo, the `sortBy` prop should be something like `user.name`. That is, if the API understands it...